features/build/buildlogic.feature

```diff
@@ -207,9 +207,9 @@ Feature: buildlogic.feature
   # @case_id OCP-13683
   Scenario: Check s2i build substatus and times
     Given I have a project
-    Given I obtain test data file "build/tc470422/application-template-stibuild.json|"
+    Given I obtain test data file "build/tc470422/application-template-stibuild.json"
```

Comment:
Should be "obtain test data dir"

Reply:
I don't think so, since it is just a file name error: could not find test file in '["/home/jenkins/workspace/Runner-v3/testdata/build/tc470422/application-template-stibuild.json|", "/home/jenkins/workspace/Runner-v3/features/tierN/testdata/build/tc470422/application-template-stibuild.json|"]'